x-pack/plugins/reporting/public/components/screen_capture_panel_content.tsx

```diff
@@ -72,43 +88,83 @@ export class ScreenCapturePanelContent extends Component<Props, State> {
             onChange={this.handlePrintLayoutChange}
             data-test-subj="usePrintLayout"
           />
-          <EuiSpacer size="s" />
-        </Fragment>
+        </EuiFormRow>
       );
     }
 
-    return (
-      <Fragment>
-        <EuiSpacer size="s" />
-      </Fragment>
-    );
+    if (this.state.isCanvasLayoutSupported) {
+      return (
+        <EuiFormRow
+          helpText={
+            <FormattedMessage
+              id="xpack.reporting.screenCapturePanelContent.canvasLayoutHelpText"
+              defaultMessage="Remove borders and footer logo"
+            />
+          }
+        >
+          <EuiSwitch
+            label={
+              <FormattedMessage
+                id="xpack.reporting.screenCapturePanelContent.canvasLayoutLabel"
+                defaultMessage="Full page layout"
+              />
+            }
+            checked={this.state.useCanvasLayout}
+            onChange={this.handleCanvasLayoutChange}
+            data-test-subj="reportModeToggle"
+          />
+        </EuiFormRow>
+      );
+    }
+
+    return null;
   };
 
   private handlePrintLayoutChange = (evt: EuiSwitchEvent) => {
-    this.setState({ usePrintLayout: evt.target.checked });
+    this.setState({ usePrintLayout: evt.target.checked, useCanvasLayout: false });
   };
 
-  private getLayout = () => {
+  private handleCanvasLayoutChange = (evt: EuiSwitchEvent) => {
+    this.setState({ useCanvasLayout: evt.target.checked, usePrintLayout: false });
```

Comment:
The UX here looks like if the user presses the canvas switch (say it is in the "off" position) it will set `usePrintLayout` to `false`. Pressing it again will leave `usePrintLayout` in `false`. I guess this leaves in in the "default" layout which would be "X"?

I'm wondering, if these controls are related in this way (cannot choose one while we've chosen the other). I'm wondering whether a button group component or similar might not be more appropriate i.t.o. UX. Open to other thoughts here!

Reply:
OK, I see I was mistaken on this one. Looking at the `private renderOptions` function I thought that we would render two toggles in some instances but then I saw the `isPrintLayoutSupported` and `isCanvasLayoutSupported` are mutually exclusive values that are determined at component mount.

IMO it would be clearer for readers if we read:

`if (this.state.isPrintLayoutSupported)` as `if (this.props.layoutOption === 'print')` and chained the if as `else if ...` for the subsequent canvas layout check to communicate their mutual exclusivity. This is a nit, won't block on this!